flake8_type_checking/checker.py

```diff
@@ -97,6 +97,8 @@ def visit(self, node: ast.AST) -> None:
         elif isinstance(node, (ast.Tuple, ast.List)):
             for n in node.elts:
                 self.visit(n)
+        elif isinstance(node, ast.Starred) and isinstance(node.ctx, ast.Load):
+            self.visit(node.value)
```

Comment:
This seems to suggest we'd support this syntax on all Python versions. Is that intentional?

Reply:
Yeah, it's the same rationale as with supporting `|` on earlier versions, invalid runtime uses will already be detected by type checkers and inside string annotations (i.e. also in regular annotations when you use `from __future__ import annotations`) it's legal to use future syntax.

In the case of starred expressions most(all?) uses in type expressions are actually invalid syntax prior to 3.11, so we have to worry even less.